### PR TITLE
Change DHCP range api and set server as default mode

### PR DESCRIPTION
**Pull request by patrickelectric**

### CODE REVIEW

File: core/libs/commonwealth/commonwealth/utils/DHCPServerManager.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ def __init__(
         interface: str,
         ipv4_gateway: IPv4Address,
         subnet_mask: Optional[IPv4Address] = None,
-        ipv4_lease_range: Optional[tuple[IPv4Address, IPv4Address]] = None,
+        lease_range: Union[int, int] = (101, 200),
```
Comment (Williangalvani) [inline core/libs/commonwealth/commonwealth/utils/DHCPServerManager.py:19]:
```suggestion
        lease_range: Tuple[int, int] = (101, 200),
```

Comment (patrickelectric) [reply core/libs/commonwealth/commonwealth/utils/DHCPServerManager.py:19]: nice

**Comment by patrickelectric** [comment]
ping @Williangalvani @JoaoMario109

**Review by JoaoMario109 — APPROVED** [review]

**Comment by goasChris** [comment]
Does this mean that blueOS would act as a DHCP server on the default eth0 network? If that is the case, then that worries me a little bit, since we usually have an existing device there that does that already, and having two DHCP servers is usually not a great experience.

Or does it only kick in if it did not already get a DHCP lease from an existing DHCP server?

**Pull request merged**